### PR TITLE
Add package makepasswd, and dependencies perl-Crypt-OpenSSL-Random, perl-Crypt-PasswdMD5

### PR DESCRIPTION
**Pull request by deathsyn**
Brings in makepasswd.

### CODE REVIEW

File: srcpkgs/makepasswd/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'makepasswd'
+pkgname=makepasswd
+version=1.10.9
+revision=1
+noarch=yes
+depends="perl-Crypt-OpenSSL-Random perl-Crypt-PasswdMD5"
+short_desc="Create and/or encrypt true-random-seeded password strings"
+maintainer="Christian Neukirchen <kevin@opensourcealchemist.com>"
```
Comment (dominikh) [inline srcpkgs/makepasswd/template]: I don't think that's your name.

**Comment by dominikh** [comment]
Please squash bc108b1 and f56688e.

**Comment by deathsyn** [comment]
Oops!  Corrected.  Sorry

**Comment by xtraeme** [comment]
Well done Barry! :+1: 

Thanks

**Pull request merged**